docs/tasks/tls/managing-tls-in-a-cluster.md

```diff
@@ -0,0 +1,129 @@
+### Managing TLS Certificates in a Cluster
+
+Every Kubernetes has a cluster root Certificate Authority (CA).  The CA is generally used by cluster components to validate the server certificate of the API server, by the API server to validate kubelet client certificates, etc. To support this, the CA certificate bundle is distributed to every node in the cluster and is distributed as a secret attached to default service accounts. A user or system component can also requrest a certificate signing using the `certificates.k8s.io` API using a protocol that is similar to the [ACME draft](https://letsencrypt.github.io/acme-spec/).
```

Comment:
We need a root CA for system components, but I will add an "Optionally" for use with user workload.